dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sfdr/categories/social/subcategories/SfdrSocialSocialAndEmployeeMatters.kt

```diff
@@ -1,81 +1,134 @@
- */
+*/
 data class SfdrSocialSocialAndEmployeeMatters(
+    @field:Valid
     val humanRightsLegalProceedings: ExtendedDataPoint<YesNo>? = null,
 
+    @field:Valid
     val iloCoreLabourStandards: ExtendedDataPoint<YesNo>? = null,
 
+    @field:Valid
     val environmentalPolicy: BaseDataPoint<YesNo>? = null,
 
+    @field:Valid
     val corruptionLegalProceedings: ExtendedDataPoint<YesNo>? = null,
 
+    @field:Valid
     val transparencyDisclosurePolicy: BaseDataPoint<YesNo>? = null,
 
+    @field:Valid
     val humanRightsDueDiligencePolicy: BaseDataPoint<YesNo>? = null,
 
+    @field:Valid
     val policyAgainstChildLabour: BaseDataPoint<YesNo>? = null,
 
+    @field:Valid
     val policyAgainstForcedLabour: BaseDataPoint<YesNo>? = null,
 
+    @field:Valid
     val policyAgainstDiscriminationInTheWorkplace: BaseDataPoint<YesNo>? = null,
 
+    @field:Valid
     val iso14001Certificate: BaseDataPoint<YesNo>? = null,
 
+    @field:Valid
     val policyAgainstBriberyAndCorruption: BaseDataPoint<YesNo>? = null,
 
+    @field:Valid
     val fairBusinessMarketingAdvertisingPolicy: ExtendedDataPoint<YesNo>? = null,
 
+    @field:Valid
     val technologiesExpertiseTransferPolicy: ExtendedDataPoint<YesNo>? = null,
 
+    @field:Valid
     val fairCompetitionPolicy: ExtendedDataPoint<YesNo>? = null,
 
+    @field:Valid
     val violationOfTaxRulesAndRegulation: ExtendedDataPoint<YesNo>? = null,
 
+    @field:Valid
     val unGlobalCompactPrinciplesCompliancePolicy: BaseDataPoint<YesNo>? = null,
 
+    @field:Valid
     val oecdGuidelinesForMultinationalEnterprisesGrievanceHandling: ExtendedDataPoint<YesNo>? = null,
 
+    @field:Valid
+    @field:DataPointMinimumValue(minimumValue = 0)
     val averageGrossHourlyEarningsMaleEmployees: CurrencyDataPoint? = null,
 
+    @field:Valid
+    @field:DataPointMinimumValue(minimumValue = 0)
     val averageGrossHourlyEarningsFemaleEmployees: CurrencyDataPoint? = null,
 
+    @field:Valid
+    @field:DataPointMinimumValue(minimumValue = -100)
+    @field:DataPointMaximumValue(maximumValue = 100)
     val unadjustedGenderPayGapInPercent: ExtendedDataPoint<BigDecimal>? = null,
 
+    @field:Valid
+    @field:DataPointMinimumValue(minimumValue = 0)
     val femaleBoardMembers: ExtendedDataPoint<Long>? = null,
 
+    @field:Valid
+    @field:DataPointMinimumValue(minimumValue = 0)
     val maleBoardMembers: ExtendedDataPoint<Long>? = null,
 
+    @field:Valid
+    @field:DataPointMinimumValue(minimumValue = 0)
+    @field:DataPointMaximumValue(maximumValue = 100)
     val boardGenderDiversityInPercent: ExtendedDataPoint<BigDecimal>? = null,
 
+    @field:Valid
     val controversialWeaponsExposure: ExtendedDataPoint<YesNo>? = null,
 
+    @field:Valid
     val workplaceAccidentPreventionPolicy: BaseDataPoint<YesNo>? = null,
 
+    @field:Valid
+    @field:DataPointMinimumValue(minimumValue = 0)
+    @field:DataPointMaximumValue(maximumValue = 100)
```

Comment:
Could the rate be higher than 100%.  Also rate per what, that at least for me is not clear reading the tooltip. Mayber Julia can give more insight here.